### PR TITLE
Miscellaneous man page updates

### PR DESCRIPTION
**Pull request by nalind**
The conversion to man pages is pretty sensitive to whitespace, so added newlines to places where we were getting unwanted line wrapping.  Fixed a couple of typos.  Expanded on a few descriptions.

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
Dang!

This was going to be be my Monday fun.  I saw the wrap issue the other
day and was going to touch it up then.  Thanks for the touch up and
happy weekend!

t

File: docs/buildah-add.md [hunk 1]
```diff
@@ -1,23 +1,31 @@
-## buildah-add "March 2017"
+## buildah-add "1" "March 2017" "buildah"
```
Comment (TomSweeneyRedHat) [inline docs/buildah-add.md]: should the 1 be in parens?  (1) or does md2man take care of that?

File: docs/buildah-rmi.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-
+buildah rmi - Removes one or more images.
 
 ## SYNOPSIS
-**buildah** **rmi** **imageID(s)** 
+**buildah** **rmi** **imageID [...]**
 
 ## DESCRIPTION
-Removes a locally stored image or images.  Multiple images are space separated.  If multiple images are passed to this command and the removal fails on one, the images following that image will not be removed.   
+Removes one or more locally stored images.
```
Comment (TomSweeneyRedHat) [inline docs/buildah-rmi.md]: Your call, but you're thinking we don't need the heads up on what happens if the removal of an image in the middle of the list isn't warranted?

File: docs/buildah.md [hunk 3]
```diff
@@ -1,18 +1,17 @@
...
 ## DESCRIPTION
 The buildah package provides a command line tool which can be used to:
 
     * Create a working container, either from scratch or using an image as a starting point.
-    * Mount the working container's root filesystem for manipulation.
-    * Unmount the working container's root filesystem.
-    * Use the updated contents of the container's root filesystem as a filesystem layer to create a new image.
-    * Delete a working container or it's image.
- 
+    * Mount a working container's root filesystem for manipulation.
```
Comment (TomSweeneyRedHat) [inline docs/buildah.md]: FWIW, I lifted most of this list from the main README.md.  You might want to touch that up too in this or another PR.

File: docs/buildah-rmi.md (hunk 2 above)
Comment (TomSweeneyRedHat) [inline docs/buildah-rmi.md]: NVM!  Just saw your other PR that changed this functionality for the better.  Ignore this comment.

File: docs/buildah-add.md (hunk 1 above)
Comment (nalind) [inline docs/buildah-add.md]: Yes, it does.

File: docs/buildah.md (hunk 3 above)
Comment (nalind) [inline docs/buildah.md]: Ah, good catch.  Updating the list in README.md to match.  Thanks!

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ d9fc0c7

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ 186aabe

**Pull request closed without merging**